README.adoc

```diff
@@ -91,13 +91,13 @@ tekton-dashboard   tekton-dashboard-tekton-pipelines.apps.kabanero.os.fyre.ibm.c
 Use the Jenkinsfile template from the reference section of this guide, and change the Docker Hub source to your own image repository. Push this Jenkinsfile to your GitHub repository in the root location.
 
 . **Create a deploy file**:
-On your local environment where Appsody is installed, go to your project folder and run the following command to generate an `app-deploy.yaml` file:
+On your local environment where your development application is installed, go to your project folder and run the following command to generate an `app-deploy.yaml` file:
 +
 ----
 appsody-deploy –-generate-only
```

Comment:
This step is now out of date.  I am thinking this whole guide needs to be refreshed.

Reply:
@kvijai82 vijai can give you details about what has to happen now.

Reply:
Hi Steve, "appsody-deploy –-generate-only" is now changed to "appsody build" per Brian. I made the update in this PR.